taxonomies/categories.txt

```diff
@@ -32376,7 +32388,11 @@ fr:Yaourts au Bifidus
 nl:Bifidus yoghurts
 
 <en:Bifidus yogurts
-fr:Yaourts au bifidus nature
+en:Sweetened plain bifidus yogurts, Yogurt type plain fermented milk with sugar with bifidus, Yogurt type plain dairy specialty with sugar with bifidus
+fr:Yaourts au bifidus nature, Lait fermenté de type yaourt nature au bifidus, spécialité laitière de type yaourt nature au bifidus
```

Comment:
the fr: entry does not match the en: entry -> en is sweetened, fr is not

Reply:
I think that one was a bad copy-paste, thanks for the info :)